modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/applicationNativeAuthentication/ApplicationNativeAuthenticationFederationTestCase.java

```diff
@@ -0,0 +1,680 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Reply:
Resolved [cb73fe4](https://github.com/wso2/product-is/pull/18629/commits/cb73fe472876fa56f3ae98730ebce83910178866)